runtime/compiler/runtime/Runtime.cpp

```diff
@@ -295,6 +295,10 @@ JIT_HELPER(interpreterUnresolvedStaticGlue);
 #ifdef J9VM_OPT_JAVA_CRYPTO_ACCELERATION
 JIT_HELPER(doAESInHardwareJit);
 JIT_HELPER(expandAESKeyInHardwareJit);
+#if defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS)
```

Comment:
At first glance these two defines seem to be mutually exclusive. Either we have compressed pointers or we don't. Am I misinterpreting the intents of these defines?

Reply:
Today, they are exclusive (in fact, FULL is defined iff COMPRESSED is not). In the future mixed mode builds, both may be defined (this is how I test).